packages/message-box/src/main.vue

```diff
@@ -10,7 +10,7 @@
       :aria-label="title || 'dialog'"
     >
       <div class="el-message-box" :class="[customClass, center && 'el-message-box--center']">
-        <div class="el-message-box__header" v-if="title !== undefined">
+        <div class="el-message-box__header" v-if="title !== null || title !== undefined">
```

Comment:
`title !== null || title !== undefined` always returns `true` regardless of what `title` is. I think it should be `title !== null` here.